test_dir/owners.txt

```diff
@@ -0,0 +1,2 @@
+RiflerRick
+
```

Comment:
RiflerRick committed on a restricted directory: test_dir/owners.txtin the repository: BB_MCS_Test belonging to Rajdeep Mukherjee(RiflerRick) authored by @RiflerRick in the file test_dir/owners.txt
Owners: no owner file found
 Collaborators: @manasisingh67, @RiflerRick,